src/components/blocks/contact/Contact.tsx

```diff
@@ -62,6 +62,7 @@ const Contact: FC = () => {
                   <NextLink title="(+91) 982 501 6079" href="tel:+919825016079" /> (India){' '}
                   <br className="d-none d-md-block" />
                   <NextLink title="(+1) 678 804 7749" href="tel:+16788047749" /> (US)
+                  <NextLink title="(+1) 706 403 8982" href="tel:+17064038982" /> (US)
```

Comment:
Put the number in new line. Currently it is showing like this 
![image](https://github.com/piyushpaliwal/nextjs-ambit/assets/635693/3b300aa6-e801-44f4-b7c1-fc76d1caa884)